README.md

```diff
@@ -383,28 +474,46 @@ Finally, we can move to part describing the new cool features we bring with `rel
       f x
   ```
 
-* `for_` for loops. There's also `forM_` but `for_` looks a bit nicer.
+* `for_` and `forM_` for loops.
 
   ```haskell
   for_ [1..10] $ \i -> do
```

Comment:
I just realized that `for_ [1..10]` is actually an antipattern, so let's replace indexed for-loop with foreach-loop...
```suggestion
  for_ files $ \file -> do
```